crates/jstz_cli/src/config.rs

```diff
@@ -136,51 +189,31 @@ pub struct SandboxConfig {
     pub octez_node_dir: PathBuf,
     /// Directory of the octez rollup node
     pub octez_rollup_node_dir: PathBuf,
-    /// The port of the octez node
-    pub octez_node_port: u16,
-    /// The port of the octez RPC node
-    pub octez_node_rpc_port: u16,
-    /// The port of the jstz node
-    pub jstz_node_port: u16,
     /// Pid of the pid
     pub pid: u32,
-    private: (),
 }
 
-impl SandboxConfig {
-    pub fn new(
-        pid: u32,
-        octez_client_dir: PathBuf,
-        octez_node_dir: PathBuf,
-        octez_rollup_node_dir: PathBuf,
-    ) -> Self {
-        Self {
-            octez_client_dir,
-            octez_node_dir,
-            octez_rollup_node_dir,
-            pid,
-            octez_node_port: 18731,
-            octez_node_rpc_port: 18730,
-            jstz_node_port: 8933,
-            private: (),
-        }
-    }
-}
+pub const SANDBOX_OCTEZ_NODE_PORT: u16 = 18731;
+pub const SANDBOX_OCTEZ_NODE_RPC_PORT: u16 = 18730;
+pub const SANDBOX_JSTZ_NODE_PORT: u16 = 8933;
+pub const SANDBOX_OCTEZ_SMART_ROLLUP_PORT: u16 = 8932;
```

Comment:
yeah, I guess it makes sense to have these hard-coded and not allowing the user to change the values.